test/Microsoft.AspNetCore.Blazor.E2ETest/Tests/BindTest.cs

```diff
@@ -47,6 +47,35 @@ public void CanBindTextbox_InitiallyPopulated()
             target.SendKeys("Changed value\t");
             Assert.Equal("Changed value", boundValue.Text);
         }
+        
+        [Fact]
+        public void CanBindTextArea_InitiallyBlank()
+        {
+            var target = Browser.FindElement(By.Id("textarea-initially-blank"));
+            var boundValue = Browser.FindElement(By.Id("textarea-initially-blank-value"));
+            Assert.Equal(string.Empty, target.GetAttribute("value"));
+            Assert.Equal(string.Empty, boundValue.Text);
+
+            // Modify target; verify value is updated
+            target.SendKeys("Changed value");
+            Assert.Equal(string.Empty, boundValue.Text); // Don't update as there's no change event fired yet.
+            target.SendKeys("\t");
+            Assert.Equal("Changed value", boundValue.Text);
+        }
+
+        [Fact]
+        public void CanBindTextArea_InitiallyPopuplated()
```

Comment:
Misprint in the title CanBindTextArea_Initially**Popuplated**
Correct writing: CanBindTextArea_Initially**Populated**

Reply:
Woops will correct that in a moment 😄 Thanks for pointing it out.